tools/image_processing/imagej2/imagej2_macros.xml

```diff
@@ -1,8 +1,9 @@
 <macros>
-    <token name="@WRAPPER_VERSION@">3.0</token>
+    <token name="@TOOL_VERSION@">20170530</token>
+    <token name="@VERSION_SUFFIX@">20170530</token>
```

Comment:
This looks wrong. But the old .1 was also not good, so no idea what is bigger.

Reply:
I just like it if the tool version matches the version of the underlying software

```python
>>> from packaging.version import Version
>>> Version("20170530+galaxy0") > Version("3.0.1")
True
```

Reply:
Bu the suffix is the same as the version?